appendices/basislz-bitstream.adoc

```diff
@@ -213,7 +213,7 @@ Any remaining bits may be safely ignored.
 [#etc1s_slice]
 === ETC1S Slice Decoding
 
-The data for each mip level is a set of ETC1S slices. The corresponding element of the imageDescs array in the <<basislz_global_data_structure>> provides slice locations within the mip level.
+The data for each mip level is a set of ETC1S slices. The corresponding element of the imageDescs array in the <<basislz_global_data_structure>> provides the location of a slice within its mip level.
```

Comment:
- Each element of the `imageDescs` array contains information about one (RGB) or two (RGB and Alpha) image slices 
- nit: Use backticks for `imageDescs`

Reply:
I fixed these 2 things.